pinecone/client.go

```diff
@@ -132,6 +400,62 @@ func (c *Client) ListIndexes(ctx context.Context) ([]*Index, error) {
 	return indexes, nil
 }
 
+// CreatePodIndexRequest holds the parameters for creating a new pods-based index.
+//
+// Fields:
+//   - Name: The name of the pods-based index to be created.
+//   - Dimension: The dimension of the index (must match [dimensionality] of upserted vectors).
+//   - Metric: The metric used to measure the [similarity] between vectors.
```

Comment:
Would add `You can use 'euclidean', 'cosine', or 'dotproduct'.`

Reply:
Ah I actually had that in a previous iteration but removed it b/c I thought w/the hyperlink it was too verbose, but I like it actually :)